joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/dataflow/DataFlowTests.scala

```diff
@@ -2518,4 +2518,23 @@ class DataFlowTests extends RubyCode2CpgFixture(withPostProcessing = true, withD
     val sink   = cpg.call.name("puts").l
     sink.reachableByFlows(source).size shouldBe 2
   }
+
+  "dataflow in method defined under class << self block" ignore {
```

Comment:
Please add a comment explaining why it's being ignored (in this case, it's the missing translation for `class << self`) so it's clear when to enable this test.

Reply:
Added comment here: https://github.com/joernio/joern/pull/3368/commits/6abf690b5d8fd398096bf358f6a79727d3b48d2d